shared/observability/src/airflow_shared/observability/metrics/otel_logger.py

```diff
@@ -28,24 +28,22 @@
 from opentelemetry.exporter.otlp.proto.http.metric_exporter import OTLPMetricExporter
 from opentelemetry.sdk.metrics import MeterProvider
 from opentelemetry.sdk.metrics._internal.export import ConsoleMetricExporter, PeriodicExportingMetricReader
-from opentelemetry.sdk.resources import HOST_NAME, SERVICE_NAME, Resource
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
 
-from airflow.configuration import conf
-from airflow.metrics.protocols import Timer
-from airflow.metrics.validators import (
+from airflow._shared.observability.metrics.protocols import Timer
```

Comment:
Those imports here should be in the form of relative imports `from .observability` - this is the way how you are able to import something relatively and reuse it in different places.